doc/releases/changelog-dev.md

```diff
@@ -19,9 +19,29 @@
   
   [(#1063)](https://github.com/PennyLaneAI/catalyst/pull/1063)
 
+* Fixes an issue where certain JAX linear algebra functions from `jax.scipy.linalg` gave
+  incorrect results when invoked from within a qjit block, and adds support for various
+  other `jax.scipy.linalg` functions, including, but not limited to:
+
+  * [`jax.scipy.linalg.cholesky`](https://jax.readthedocs.io/en/latest/_autosummary/jax.scipy.linalg.cholesky.html)
```

Comment:
Very minor, but I would find it slightly more readable if we use a different character for enumeration than at the top-level:
```suggestion
  - [`jax.scipy.linalg.cholesky`](https://jax.readthedocs.io/en/latest/_autosummary/jax.scipy.linalg.cholesky.html)
```

Reply:
Fixed